src/e2e/features/7-taken-verdelen.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Zaken verdelen
+
+  Scenario: Bob wants to verdeel a number of taken
```

Comment:
We are using the term 'assign' in the backend instead of verdelen.. This is what you get without a common vocabulary I guess. ;-) I suggest to use 'assign' here too.